src/submitScore.js

```diff
@@ -1,12 +1,13 @@
 import getScores from './getscores.js';
-import Submition from './submition.js';
 import list from './list.js';
 
-const submitScore = () => {
-  const submition = new Submition();
-  const scores = getScores;
-  scores.push([submition.name, submition.score]);
-  localStorage.setItem('scores', JSON.stringify(scores));
+const submitScore = async (url, submition) => {
+  await fetch(url, {
+    method: 'POST',
+    body: JSON.stringify(submition),
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+  });
+  const scores = await getScores(url);
```

Comment:
- [OPTIONAL] It will be good to empty the input form fields after the submit button is clicked, this will make it easier for the user to add another entry without deleting their previous entry themselves.

Reply:
Thank you for your recommendation, I agree with you and I will implement it in the next feature branch where I will be polishing the UI a little